.gitignore

```diff
@@ -28,3 +28,8 @@ simtools-output
 # Dynamic version file
 # generated by setuptools_scm
 simtools/_version.py
+
+# Coverage files
+coverage*
+.coverage*
+.coverage
```

Comment:
Is this line needed? Probably covered by `.coverage*`

Reply:
Removed now